spec/factories/comments.rb

```diff
@@ -1,8 +1,9 @@
 FactoryGirl.define do
   factory :comment do
```

Comment:
A cool thing you can do on factories for models that are always used in conjunction with other models is add a 'transient' feature. This makes it easier to associate to the commentable object (see my comment on spec/models/job_post_spec.rb)

```
transient do
   commentable
end

after
```

As an example you can look at [tag factory](https://github.com/BlackTechMecca/EcosystemNetwork/blob/master/spec/factories/tags.rb)